blocks/ipo/ipo.js

```diff
@@ -0,0 +1,183 @@
+import {
+  createPictureElement, fetchData,
+  // eslint-disable-next-line no-unused-vars
```

Comment:
do we need this ?

Reply:
yes , actually PR is feching data from mock to show difference.
will switch back to api before merge then these comment will be removed.